### PR TITLE
Book detail IV - Badge

### PR DESCRIPTION
**Pull request by seromarin**
## Summary
- Book detail badge added
- **Minor HTML / CSS readability fixes**

## Screenshot
Captura de Pantalla 2020-10-30 a la(s) 4 17 44 p  m

## Trello Card
https://trello.com/c/WJVtydRj/29-detalle-de-libro-iv-badge

### CODE REVIEW

File: maquetacion/src/scss/_detail.scss [hunk 1]
```diff
@@ -1,29 +1,91 @@
...
+}
+
+%text-content {
+  font-size: 20px;
+  color: $grey-light;
+}
+
 .detail {
-  box-shadow: 1px 10px 30px 1px rgba(0,0,0,0.2);
+  background-color: #FFF;
```
Comment (mcallegari10) [inline maquetacion/src/scss/_detail.scss:14]: Use variable for colors

**Review by mcallegari10 — APPROVED** [review]
Check the color variables and it's great! 😄

File: maquetacion/src/scss/_detail.scss (hunk 1 above)
Comment (seromarin) [reply maquetacion/src/scss/_detail.scss:14]: Fixed in last branch / commit

File: maquetacion/src/scss/_detail.scss [hunk 2]
```diff
@@ -1,29 +1,91 @@
...
+  }
+
   &__cover {
-    margin-right: 3rem;
+    margin-right: 52px;
+    position: relative;
+  }
+
+  &__cover-badge {
+    left: 200px;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_detail.scss]: use right property

File: maquetacion/src/scss/_styles.scss [hunk 3]
```diff
@@ -6,18 +6,13 @@
...
-.text-bold {
-  font-weight: 700;
-}
-
-.text-20 {
-  font-size: 1.25rem;
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (idcuadrowolox) [inline maquetacion/src/scss/_styles.scss:14]: sort properties by name

Comment (seromarin) [reply maquetacion/src/scss/_styles.scss:14]: Properties sorted

**Review by seromarin — COMMENTED** [review]
All CSS changes are made

**Review by idcuadrowolox — APPROVED** [review]

**Pull request merged**